lib/common.sh

```diff
@@ -0,0 +1,248 @@
+#! /bin/bash
+
+APPNAME="aznfs"
+RUNDIR="/run/${APPNAME}"
+OPTDIR="/opt/microsoft/${APPNAME}"
+LOGFILE="${OPTDIR}/${APPNAME}.log" 
+
+# 
+# This stores the map of local IP and share name and external blob endpoint IP. 
+# 
+MOUNTMAP="${RUNDIR}/mountmap"
```

Comment:
actually I'm thinking we should go back to storing MOUNTMAP in OPTDIR instead of RUNDIR.
There we have the chattr protection which would avoid most of the common issues with someone accidentally deleting the file.
add_mountmap() and delete_mountmap() should be the only one updating mountmap and so the chattr will be needed only in those files.